x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/setup_guide/setup_guide.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiSpacer, EuiTitle, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+import { ENTERPRISE_SEARCH_PLUGIN } from '../../../../../common/constants';
+import { SetupGuide as SetupGuideLayout } from '../../../shared/setup_guide';
+import { SetEnterpriseSearchChrome as SetPageChrome } from '../../../shared/kibana_chrome';
+import { SendEnterpriseSearchTelemetry as SendTelemetry } from '../../../shared/telemetry';
+import GettingStarted from './assets/getting_started.png';
+
+export const SetupGuide: React.FC = () => (
+  <SetupGuideLayout
+    productName={ENTERPRISE_SEARCH_PLUGIN.NAME}
+    productEuiIcon="logoEnterpriseSearch"
+    standardAuthLink="https://www.elastic.co/guide/en/app-search/current/security-and-users.html#app-search-self-managed-security-and-user-management-standard"
+    elasticsearchNativeAuthLink="https://www.elastic.co/guide/en/app-search/current/security-and-users.html#app-search-self-managed-security-and-user-management-elasticsearch-native-realm"
+  >
+    <SetPageChrome
+      text={i18n.translate('xpack.enterpriseSearch.setupGuide.title', {
+        defaultMessage: 'Setup Guide',
+      })}
+    />
+    <SendTelemetry action="viewed" metric="setup_guide" />
```

Comment:
It looks like we use the same metric for all setup guides. Do we need to distinguish?

Reply:
Because this is `SendEnterpriseSearchTelemetry` I believe its scoped to there and we only have one setup guide per telemetry scope (aka per app)

Reply:
Looks like you've answered this below in https://github.com/elastic/kibana/pull/78233#discussion_r493831997 - but yeah, these do increment different telemetry stats. The `metric` and `actions` are the same but the `product` key is what's different (aliased in the import as you noted - we can discuss that in that thread for sure).

My thinking was that I didn't think that distinction would really matter to most devs - it's mostly setup / boilerplate and not something that needs deep thinking. If you open a setup guide, you should confirm it has `<SendTelemetry />` and `<SetPageChrome />` components, but they're very similar between guides, it's just the product/alias that changes.